l10n_ar_ux/models/account_move_line.py

```diff
@@ -1,20 +0,0 @@
-from odoo import fields, models, api
-
-
-class AccountMoveLine(models.Model):
-    _inherit = 'account.move.line'
-
-    @api.onchange('currency_id', 'amount_currency')
-    def _onchange_currency(self):
```

Comment:
In this case, we do not need anymore to ensure that the subtotal are set using the already set l10n_ar_currency_rate instead of the currency actual rate?

Reply:
that functionality is deprecated for now and we'll re add later